UWP/UWP.vcxproj.filters

```diff
@@ -327,48 +429,81 @@
     <None Include="Content\flash0\font\ltn15.pgf">
       <Filter>Content\flash0\font</Filter>
     </None>
-    <None Include="Content\shaders\natural.fsh">
-      <Filter>Content\shaders</Filter>
-    </None>
-    <None Include="Content\shaders\naturalA.fsh">
-      <Filter>Content\shaders</Filter>
+    <None Include="Content\ppge_atlas.meta">
+      <Filter>Content</Filter>
     </None>
-    <None Include="Content\shaders\scanlines.fsh">
-      <Filter>Content\shaders</Filter>
+    <None Include="Content\ui_atlas.meta">
+      <Filter>Content</Filter>
     </None>
-    <None Include="Content\shaders\sharpen.fsh">
-      <Filter>Content\shaders</Filter>
+    <None Include="Content\themes\defaultthemes.ini">
+      <Filter>Content\themes</Filter>
     </None>
-    <None Include="Content\shaders\upscale_spline36.fsh">
-      <Filter>Content\shaders</Filter>
+    <None Include="Content\asciifont_atlas.meta">
+      <Filter>Content</Filter>
     </None>
-    <None Include="Content\shaders\vignette.fsh">
-      <Filter>Content\shaders</Filter>
+    <None Include="Content\font_atlas.meta">
+      <Filter>Content</Filter>
     </None>
-    <None Include="Content\shaders\videoAA.fsh">
-      <Filter>Content\shaders</Filter>
+    <None Include="Content\sfx_back.wav">
+      <Filter>Content</Filter>
     </None>
-    <None Include="..\assets\ppge_atlas.meta">
+    <None Include="Content\sfx_confirm.wav">
       <Filter>Content</Filter>
     </None>
-    <None Include="..\assets\ui_atlas.meta">
+    <None Include="Content\sfx_select.wav">
       <Filter>Content</Filter>
     </None>
-    <None Include="..\assets\themes\defaultthemes.ini">
-      <Filter>Content\themes</Filter>
+    <None Include="Content\sfx_toggle_off.wav">
+      <Filter>Content</Filter>
     </None>
-    <None Include="Content\asciifont_atlas.meta">
+    <None Include="Content\sfx_toggle_on.wav">
       <Filter>Content</Filter>
     </None>
+    <None Include="Content\shaders\5xBR-lv2.fsh">
+      <Filter>Content\shaders</Filter>
+    </None>
+    <None Include="Content\shaders\natural.vsh">
+      <Filter>Content\shaders</Filter>
+    </None>
+    <None Include="Content\debugger\.nojekyll">
+      <Filter>Content\debugger</Filter>
+    </None>
+    <None Include="Content\debugger\asset-manifest.json">
+      <Filter>Content\debugger</Filter>
+    </None>
+    <None Include="Content\debugger\index.html">
+      <Filter>Content\debugger</Filter>
+    </None>
+    <None Include="Content\debugger\manifest.json">
+      <Filter>Content\debugger</Filter>
+    </None>
+    <None Include="Content\debugger\static\css\main.3eab8a01.css">

```

Comment:
Ah, it's unfortunate if the only way is to include each filename in the vcxproj.  These filenames will by default change every time there's an update.

-[Unknown]

Reply:
You can thank how vcxproj files wanted to work, there's no easy workaround for that sadly.

Reply:
Actually, I think I have an idea of how to fix. I'll push it soon.